content/docs/introduction/autoscaling.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Autoscaling
+subtitle: Learn how Neon automatically and transparently scales compute on demand
+enableTableOfContents: true
+---
+
+A Beta version of Neon's _Autoscaling_ feature is now available for [paid plan](https://neon.tech/docs/introduction/billing#neon-plans) users in selected regions. The Autoscaling feature dynamically adjusts the allocation of vCPU and RAM for a Neon compute endpoint in response to the current load, eliminating the need for manual intervention.
+
+## Autoscaling benefits
+
+Neon's Autoscaling feature offers the following benefits:
+
+- **On-demand scaling:** Autoscaling helps with workloads that experience variations over time, such as applications with region or time-based changes in demand.
+- **Cost effectiveness**: Autoscaling optimizes resource utilization, ensuring that organizations only pay for required resources, rather than over-provisioning to handle peak loads.
+- **Resource and cost control**: Autoscaling operates within a user-defined range, ensuring that your compute resources and associated costs do not scale indefinitely.
+- **No manual intervention**: After you enable Autoscaling and set scaling limits, no manual intervention is required, allowing you to focus on your applications.
+
+## How to enable Autoscaling
+
+You can enable Autoscaling when creating a Neon project or afterward using a simple compute configuration dialog. If you have multiple computes, you can enable Autoscaling for each compute individually. For instructions, see:
```

Comment:
I don't know the current state of things; IIRC "enable for a single compute" is still in progress (cc @tychoish?)

Reply:
This isn't released yet.

Reply:
It's not on prod yet. Removed.